packages/UI/src/components/atoms/Avatar/Avatar.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { tv, type VariantProps } from 'tailwind-variants';
+
+const img = tv({
+  base: 'w-16 h-16 bg-green-500 flex justify-center items-center text-white  rounded  font-medium',
```

Comment:
if you want to use radix ui maybe you should switch base to slots
https://www.tailwind-variants.org/docs/slots